packages/vitest/src/node/error.ts

```diff
@@ -213,12 +200,11 @@ function printStack(
 export function generateCodeFrame(
   source: string,
   indent = 0,
-  start: number | Position = 0,
-  end?: number,
+  pos: Position,
```

Comment:
Oh. This is a stack, not a position. Let's also remove `Position` type, because we don't set it anywhere, and it uses `source` property instead of `file`, which might be confusing.

Reply:
Yeah, I thought the same, but was wary of removing Position as it was used somewhere else. Just refactored `Position` out 🙇‍♂️!